### PR TITLE
fix: npm badges and dist tags

### LINKED ISSUE #34: Bug in badges with scoped packages
Because the "workaround" https://github.com/amio/badgen-service/blob/master/libs/live-fns/npm.js#L96

It seems that `https://registry.npmjs.org/@babel%2Fcore/*` shows empty object
but removing the star shows all versions. 

proof: https://registry.npmjs.org/@babel%2Fcore

It's interesting why with `@nestjs/core/*` it still works :laughing:

### PR DESCRIPTION
**Pull request by olstenlarck**
fix: resolves #34 and resolves #35

### CODE REVIEW

**Comment by olstenlarck** [comment]
2018-07-30-07 17 56_1280x1024_scrot

**Comment by olstenlarck** [comment]
2018-07-30-07 31 55_1280x1024_scrot

File: libs/live-fns/npm.js [hunk 1]
```diff
@@ -33,25 +33,64 @@ module.exports = async function npm (method, ...args) {
...
+    }
   }
 }
 
+pkg('version', ['ava'])
+pkg('version', ['ava', 'next'])
+pkg('version', ['next', 'canary'])
+pkg('version', ['babel-core'])
+pkg('version', ['@babel', 'core'])
+pkg('version', ['@nestjs', 'core', 'beta'])
```
Comment (amio) [inline libs/live-fns/npm.js]: Is these for testing?

Comment (olstenlarck) [reply libs/live-fns/npm.js]: Damn. I forgot them ;d But yea, that way i test the things :D

**Pull request merged**